### PR TITLE
Reference nested placeables

### PR DESCRIPTION
**Pull request by zbraniecki**
Fixes nested placeables for #239

### CODE REVIEW

**Comment by stasm** [comment]
Hey @zbraniecki, thanks for the patch. The change looks good, but it's a bit hard to review without at least one test testing the desired parse output. Also, for small changes like this basing the PR on top of `feature/syntax-0.6` would make it easier to review and land. Can you make these changes, please?

**Comment by zbraniecki** [comment]
> The change looks good, but it's a bit hard to review without at least one test testing the desired parse output.

My assumption is that we're landing this to enable reference tests which will replace our current tests. Do you want to duplicate tests between them?

**Comment by stasm** [comment]
Yes, at least one test please. I'm still not sure we'll be able to unskip the reference fixtures.

File: fluent-syntax/test/fixtures_behavior/placeable_in_placeable.ftl [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #   }
```
Comment (stasm) [inline fluent-syntax/test/fixtures_behavior/placeable_in_placeable.ftl]: Can we un-comment `key1`-`key3` now?

**Comment by zbraniecki** [comment]
Hmm, `key1` and `key2` - yes, for `key3` - no, and it seems to me that it should not work based on https://github.com/projectfluent/fluent/blob/master/syntax/grammar.mjs#L162 - it only allows for inline ws between first `{` and second `{`. Did I miss something?

**Comment by stasm** [comment]
No, you're right, thanks. `key3` is related to https://github.com/projectfluent/fluent/issues/106 which is planned for Syntax 0.7.

**Pull request merged**